### ISSUE TITLE
Update native MongoDB driver

### ISSUE THREAD

**Issue by simison** [Trustroots/trustroots#753]
[`mongodb`](https://www.npmjs.com/package/mongodb) driver needs updating from v2 to v3.

https://github.com/Trustroots/trustroots/blob/9574266d8d1a04bde96f1093b12e208af2205126/package.json#L114

See breaking changes:

https://github.com/mongodb/node-mongodb-native/blob/HEAD/CHANGES_3.0.0.md

It's used by Worker script, here: 
https://github.com/Trustroots/trustroots/blob/15a8bcebee8286eca1591f950a120420b2e08dd3/config/lib/worker.js

Shouldn't affect the main app script (app and worker run separately in parallel with `npm start`), since the app script depends on Mongoose handling the connection directly.

Alternatively we could refactor Worker depend on the Mongoose connection just like the main app script does. That would require changes to Angeda configuration: https://github.com/agenda/agenda#mongomongoclientinstance

**Comment by steaward** [comment]
Hey, just caught wind of this project. I'd love to help contribute. 
I can attempt to tackle this issue for you.

**Comment by simison** [comment]
Yay! That would be amazing, @steaward

**Comment by steaward** [comment]
@simison it seems that I am able to use mongoDB v 3.1.8 (this is the version I see on NPM) with Trustroots. 

I ran the script `scripts/fillTestData.js` as well as `scripts/fillTestTribesData.js` and things seemed to work without problems. 

Something did pop up though:
When I ran `npm test`, 4 tests failed. However, the `master` is also failing these same 4 (they are to due with changing a user's profile picture to jpeg, gif, ect.). Is this just me failing somewhere, or is this a known issue?

Just a little concerned before I make a PR. 
Thanks :)
